### PR TITLE
Operation protection for VMFS cmdlets

### PR DESCRIPTION
**Pull request by srajmohan24**
The changes in this PR are as follows:

* Add functions to verify signature of a VMFS Run Command API request.
* Update documentation to specify the necessary steps to generate a signature.

I have read the [contributor guidelines](CONTRIBUTING.md) and have completed the following:

* [x] **Formatted the code** using VSCode default formatter for PowerShell.
* [ ] **Tested the code** end-to-end against an SDDC.
* [x] **Documented the functions** using standard PowerShell markup and applied `AVSAttribute` to newly exported functions.

### CODE REVIEW

File: Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1 [hunk 1]
```diff
@@ -69,12 +69,12 @@ function Set-VmfsIscsi {
         throw "Invalid SCSI IP address $ScsiIpAddress provided."
     }
 
-    $OperationProtectionSource = Get-Content -Path ".\OperationProtection.cs"
+    $OperationProtectionSource = Get-Content -Path "..\Microsoft.AVS.Management\OperationProtection.cs"
```
Comment (et1975) [inline Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1:72]: This is not going to work when we install the module, it needs to be embedded. Something like [this](https://blog.stefan-gossner.com/2010/05/07/using-csharp-c-code-in-powershell-scripts/)?

Comment (srajmohan24) [reply Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1]: Sounds good, I've added it as embedded in the VMFS psm1 file. I was initially thinking of adding the embedded code in a separate OperationProtection.ps1 file under the Microsoft.AVS.Management folder and source it in VMFS module similar to https://github.com/Azure/Microsoft.AVS.Management/blob/373858f98d61caa668ef8212b48a511f7e6ab6a5/Microsoft.AVS.Management/Microsoft.AVS.Management.psm1#LL2C30-L2C30, but I wasn't sure if the VMFS module file would have access to the other folders when installing the module?

File: Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1 [hunk 2]
```diff
@@ -1,5 +1,64 @@
 using module Microsoft.AVS.Management
 
+$OperationProtectionSource = @"
+using System;
+using System.IO;
+using System.Collections;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace OperationProtection
```
Comment (et1975) [inline Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1:10]: I think the namespace should be `Microsoft.AVS.Management`.

File: Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1 (hunk 1 above)
Comment (srajmohan24) [reply Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1]: Sounds good, I've added the embedded code in the Microsoft.AVS.Management module instead

**Review by et1975 — DISMISSED** [review]

**Pull request closed without merging**